wolfssl/internal.h

```diff
@@ -1401,7 +1401,13 @@ int  SetCipherList(Suites*, const char* list);
     WOLFSSL_LOCAL int NetX_Receive(WOLFSSL *ssl, char *buf, int sz, void *ctx);
     WOLFSSL_LOCAL int NetX_Send(WOLFSSL *ssl, char *buf, int sz, void *ctx);
 #endif /* HAVE_NETX */
-
+#ifdef HAVE_LWIP_NATIVE
+    WOLFSSL_LOCAL int wolfSSL_LwIP_Send(WOLFSSL* ssl, char *buf, int sz, void *cb);
```

Comment:
Why is this API WOLFSSL_LOCAL?  If it has any value wouldn't others might want to use it as their callback?  The above examples NetX... are WOLFSSL_LOCAL because they are automatically installed as the callbacks in NetX case.  I don't see that here.